project-koku/koku
# [COST-5213] - fix S3 prepare

## Jira Ticket

[COST-5213](https://issues.redhat.com/browse/COST-5213)

## Description

This change will revert the default of `s3_parquet_cleared` flag. Then during daily processing we should set this flag accordingly.

The problem before is each worker was flipping this flag back and forth instead of holding a constant for all running workers. Meaning as we processed more files for a single provider this filtering/collection of files for deletion would take longer and longer while ultimately deleting nothing. 

## Testing

1. Checkout Branch
2. Restart Koku with multiple workers
3. Load AWS/Azure data with finalised bills which have multiple report files (X2)
4. Watch the logs and see ` removed s3 files and marked manifest s3_parquet_cleared` is only called once per month. 

## Release Notes

```markdown
* [COST-5213](https://issues.redhat.com/browse/COST-5213) Improve collection for delete from S3
```
